src/vs/workbench/services/untitled/common/untitledTextEditorModel.ts

```diff
@@ -67,7 +67,7 @@ export interface IUntitledTextEditorModel extends ITextEditorModel, ILanguageSup
 	resolve(): Promise<void>;
 }
 
-export class UntitledTextEditorModel extends BaseTextEditorModel implements IUntitledTextEditorModel {
+export class UntitledTextEditorModel extends BaseTextEditorModel implements IUntitledTextEditorModel, IWorkingCopy {
```

Comment:
Not needed, `IUntitledTextEditorModel` already implements `IWorkingCopy`?

Reply:
It made the error more understandable when I was missing properties, but I can take it back out if that's preferred